HarryPotter/Screens/House/View/HouseMemberCell.swift

```diff
@@ -17,6 +20,19 @@ final class HouseMemberCell: UITableViewCell {
         return label
     }()
     
+    lazy var myButton: UIButton = {
+        let button = UIButton(type: .custom)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.setImage(UIImage(named: "whiteheart"), for: .normal)
+        button.setImage(UIImage(named: "redheart"), for: .selected)
+        button.addTarget(self, action: #selector(didTapButton), for: .touchUpInside)
+        return button
+    }()
+    
+    @objc func didTapButton() {
+        let wasSelected = myButton.isSelected
+        myButton.isSelected = !wasSelected
```

Comment:
Yep, I think you could tell the ViewController that it was tapped here. `markedFavourite(wasSelected)`